Shared/Utils.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Utils.swift
+//  Client
+//
+//  Created by Atul Aggarwal on 8/8/15.
+//  Copyright (c) 2015 Mozilla. All rights reserved.
+//
```

Comment:
This should be an MPL2.0 license header; see the other files in the directory. Be careful to restore the spacing that Xcode will lose when you paste!